jetstream/config.py

```diff
@@ -62,64 +62,81 @@
 @attr.s(auto_attribs=True)
 class Platform:
     config_spec_path: PathLike
-    metrics_module: Optional[ModuleType]
-    segments_module: Optional[ModuleType]
     enrollments_query_type: str
     # app_id to use to validate Outcomes.
     validation_app_id: str
+    metrics_module: Optional[ModuleType] = attr.ib(
+        default=None
+    )  # kik-kik: moved to the end since this is "optional"
+    segments_module: Optional[ModuleType] = attr.ib(
+        default=None
+    )  # kik-kik: moved to the end since this is "optional"
+
+
+# TODO: the below two vars can now be extracted into a separate config file
+CONFIG_DIRECTORY = Path(__file__).parent / "config"
+PLATFORM_CONFIGS_RAW = {  # TODO: after it will be extracted into a separate toml file
+    "firefox_desktop": {
+        "config_spec_path": CONFIG_DIRECTORY / "default_metrics.toml",
+        "metrics_module": mozanalysis.metrics.desktop,
+        "segments_module": mozanalysis.segments.desktop,
+        "enrollments_query_type": "normandy",
+        "validation_app_id": "firefox-desktop",
+    },
+    "fenix": {
+        "config_spec_path": CONFIG_DIRECTORY / "fenix.toml",
+        "metrics_module": mozanalysis.metrics.fenix,
+        "segments_module": None,
+        "enrollments_query_type": "glean-event",
+        "validation_app_id": "org.mozilla.fenix",
+    },
+    "firefox_ios": {
+        "config_spec_path": CONFIG_DIRECTORY / "firefox_ios.toml",
+        "metrics_module": mozanalysis.metrics.firefox_ios,
+        "segments_module": None,
+        "enrollments_query_type": "glean-event",
+        "validation_app_id": "org.mozilla.ios.FirefoxBeta",
+    },
+    "focus_android": {
+        "config_spec_path": CONFIG_DIRECTORY / "focus_android.toml",
+        "metrics_module": mozanalysis.metrics.focus_android,
+        "segments_module": None,
+        "enrollments_query_type": "glean-event",
+        "validation_app_id": "org.mozilla.focus",
+    },
+    "klar_android": {
+        "config_spec_path": CONFIG_DIRECTORY / "klar_android.toml",
+        "metrics_module": mozanalysis.metrics.klar_android,
+        "segments_module": None,
+        "enrollments_query_type": "glean-event",
+        "validation_app_id": "org.mozilla.klar",
+    },
+    "focus_ios": {
+        "config_spec_path": CONFIG_DIRECTORY / "focus_ios.toml",
+        "metrics_module": mozanalysis.metrics.focus_ios,
+        "segments_module": None,
+        "enrollments_query_type": "glean-event",
+        "validation_app_id": "org.glean-event.focus",
```

Comment:
```suggestion
        "validation_app_id": "org.mozilla.ios.Focus",
```
This should fix the failing integration test. Right now it's trying to access a table that doesn't exist.